src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/ConsumerGroup/RemoveAzureEventHubConsumerGroups.cs

```diff
@@ -14,41 +14,82 @@
 
 using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
 using System.Management.Automation;
+using Microsoft.Azure.Commands.EventHub.Models;
+
 namespace Microsoft.Azure.Commands.EventHub.Commands.ConsumerGroup
 {
     /// <summary>
     /// 'Remove-AzureRmEventHubConsumerGroup' deletes the specifed Consumer Group
     /// </summary>
-    [Cmdlet(VerbsCommon.Remove, ConsumerGroupVerb, SupportsShouldProcess = true)]
+    [Cmdlet(VerbsCommon.Remove, ConsumerGroupVerb, DefaultParameterSetName = ConsumergroupPropertiesParameterSet, SupportsShouldProcess = true)]
```

Comment:
@v-Ajnava  [same comment made in the ServiceBus PR](https://github.com/Azure/azure-powershell/pull/6732#discussion_r204581997) -- would you mind making the output type `bool`?

Reply:
resolved